test/unit/style-spec/fixture/style-api.input.json

```diff
@@ -56,5 +56,6 @@
     "owner": "Cyrus",
     "visibility": "private",
     "cacheControl": "s-max-age=200",
-    "draft": true
+    "draft": true,
+    "protected": true
```

Comment:
@mbullington can you add a new fixture with an invalid protected value?

Reply:
Done! Added a `bad-style-api`